cmd/config_git_remove.go

```diff
@@ -25,7 +25,7 @@ func NewConfigGitRemoveCmd(newClient ClientFactory) *cobra.Command {
 	such as local generated Pipelines resources and any resources generated on the cluster.
 	`,
 		SuggestFor: []string{"rem", "rmeove", "del", "dle"},
-		PreRunE:    bindEnv("path", "namespace", "delete-local", "delete-cluster"),
+		PreRunE:    bindEnv("path", "delete-local", "delete-cluster"),
```

Comment:
the ns comes from?

Reply:
This is just cleanup.  That command does not use namespace.  This command removes git settings from the function's metadata.  This is not `func delete` which actually removes the running service (and requires namespace)